tests/api/api/feature_store/base.py

```diff
@@ -60,3 +60,63 @@ def _assert_diff_as_expected_except_for_specific_metadata(
         expected_object, actual_object, ignore_order=True, exclude_paths=exclude_paths,
     )
     assert diff == expected_diff
+
+
+def _test_group_by_for_feature_store_objects(
+    client: TestClient, object_name, project_name, count
+):
+    _list_and_assert_objects(client, object_name, project_name, None, count * 3)
+    results = _list_and_assert_objects(
+        client,
+        object_name,
+        project_name,
+        "group-by=name&sort-by=updated&rows-per-group=1&order=desc",
+        count,
+    )[object_name]
+
+    for result_object in results:
+        assert result_object["metadata"]["tag"] == "newest"
+
+    results = _list_and_assert_objects(
```

Comment:
Add some comments before each case cause it's hard to see by eye what has changed, for example here I would add `"# testing desc`